file.go

```diff
@@ -1,37 +1,174 @@
 package jcsv
 
-import "os"
+import (
+	"encoding/csv"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strconv"
+	"strings"
+)
 
 type file struct {
-	// TODO: define variables if needed
+	Data map[string]interface{}
 }
 
 func ParseJsonFile(path string) (file, error) {
-	// TODO: open and read the given file into your `file` object
-	return file{}, nil
+	//open JSON file
+	jsonFile, err := os.Open(path)
+
+	//if an error occurred
+	if err != nil {
+		fmt.Println(err)
+		return file{}, err
+	}
+
+	//defer the closing of file till the function ends
+	defer jsonFile.Close()
```

Comment:
That is what the code says.

```suggestion
	}
	defer jsonFile.Close()
```